### ISSUE TITLE
How to handle the case that the predicting width/height is bigger than the width/height of network output when training?

### ISSUE THREAD

**Issue by AlexLuya** [ultralytics/yolov5#788]
Hello,this problem comes up in first iteration
```
Is this normal? 
If yes,should I clip them to fit the size of network output?
or discard this prediction? 
```

**Comment by AlexLuya** [comment]
Thanks,this is problem comes up when training yolo v5.This code base is too hard to follow(too many jumps,hard codings in it,sorry to say this),so I am trying to understand it by debugging,and encountered this problem.

**Comment by henryle97** [comment]
Do you mean width height of network input ?

**Comment by AlexLuya** [comment]
No,I mean the size of network output is 76x76,and the predicting bbox value : c_x , c_y , w , h, and both c_x and c_y are smaller than 76,but some values of W/H are far bigger than 76,even > 300

**Comment by Ownmarc** [comment]
I would suggest you start by reading about the basics of Yolo before opening this kind of issue.
